osu.Game/Screens/Play/Player.cs

```diff
@@ -46,6 +46,8 @@ public class Player : OsuScreen
 
         public bool HasFailed { get; private set; }
 
+        public bool AllowPause { get; set; } = true;

```

Comment:
But why? Why disallow pausing in visual tests?

Reply:
switching between ruleset (step) tests doesn't work. also exiting out of the visual test takes manual mouse interaction which is really annoying.